src/components/sections/about/translations/en.json

```diff
@@ -1,13 +1,10 @@
 {
-  "about": "ABOUT",
-  "platinum": "PLATINUM",
+  "aboutPlatinum": "ABOUT <span class=\"title _orange\">PLATINUM</span>",
```

Comment:
From a BEM point of view, this is correct. The tag must not contain styles. All styles should be spelled out for classes.
But from another point of view.
If will continue to be used CMS and visual editor. Then ordinary people will have to write classes. It is not right. Therefore, if the tags p,h1-h6 tags contain tags strong, span, b, i tags for them, you do not need to add classes. Can you fix it everywhere?

Reply:
Good point. Will change it when translation work will be scheduled